plugins/module_utils/network/sonic/config/vlans/vlans.py

```diff
@@ -300,20 +323,50 @@ def get_create_vlans_requests(self, configs):
         if not configs:
             return requests
         for vlan in configs:
-            vlan_id = vlan.get("vlan_id")
+            vlan_id = vlan.get("vlan_id", None)
```

Comment:
Here and elsewhere: If the desired default "get" value is None, it isn't necessary to explicitly add the default since that is the Python default for a "get". (It is, however necessary to specify it if the desired default is, for example, an empty list or an empty string.)

```suggestion
            vlan_id = vlan.get("vlan_id")
```

Reply:
I hear you in that it is not necessary to specify None during a get() request for a dictionary. However, this file has multiple references like this so to keep in line with prior style I explicitly mentioned it here. I can remove them which for aesthetics  but that shouldn't change any functionality.

Reply:
Okay. I will drop this Issue. It is common enough in Python to do this, even though it is unnecessary. I agree that there is no functional impact and understand that this was the precedent set in this file, so it can stay "as is".